engine/tester/config_parse.c

```diff
@@ -1011,6 +1011,8 @@ get_script(xmlNodePtr node, tester_cfg *cfg, test_script *script)
 {
```

Comment:
doc/xsd/tester_defs.xsd must be fixed

Reply:
Why is it marked as resolved while I see no appropriate changes in neither the first nor the second Tester patch.

Reply:
Excuse me, I forgot to push it(